### PR TITLE
Ins 1446 zerolog adapter

### PR DESCRIPTION
**Pull request by ruz**

### CODE REVIEW

File: log/zerolog.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	switch strings.ToLower(cfg.Formatter) {
+	case "text":
+		output = zerolog.ConsoleWriter{Out: os.Stderr, NoColor: true, TimeFormat: timestampFormat}
+	case "json":
+		output = os.Stderr
+	default:
+		return nil, errors.New("unknown formatter " + cfg.Formatter)
+	}
+
+	zerolog.CallerSkipFrameCount = 4
```
Comment (nordicdyno) [inline log/zerolog.go]: Could you add some comment with description why 4 or link on doc where this description present?

File: log/log.go [hunk 2]
```diff
@@ -28,6 +28,7 @@ import (
 )
 
 const defaultSkipCallNumber = 3
+const timestampFormat = "2006-01-02 15:04:05.000000"
```
Comment (nordicdyno) [inline log/log.go]: Why custom format? Why not  RFC 3339?

File: log/log.go [hunk 3]
```diff
@@ -28,6 +28,7 @@ import (
 )
 
 const defaultSkipCallNumber = 3
```
Comment (nordicdyno) [inline log/log.go]: why 3?

Comment (ruz) [reply log/log.go]: added test to show that it is correct

File: log/log.go (hunk 2 above)
Comment (ruz) [reply log/log.go]: we want sub seconds

File: log/zerolog.go (hunk 1 above)
Comment (ruz) [reply log/zerolog.go]: no doc, no description, added test to show that it is correct

**Pull request merged**